src/pyOpenMS/tests/unittests/run_all_split_tests.sh

```
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Test runner for all split test files
+# Part of Issue #8567 - test000.py splitting
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)